config/crds/kudo.dev_instances.yaml

```diff
@@ -120,7 +120,7 @@ spec:
                   \      v                       v | Fatal error |        |    Complete
                   \   |"
                 properties:
-                  lastFinishedRun:
+                  lastUpdated:
```

Comment:
About naming: everywhere esle in k8s world these fields are called `*Timestamp`, e.g. `creationTimestamp`, `deletionTimestamp` etc. I think we should call it `updateTimestamp` to keep it homogenous with the rest. wdyt @yankcrime ?

Reply:
I like adding Timestamp especially for consistency's sake, I think `updateTimestamp` could be a bit ambiguous at first glance though so I've gone with `lastUpdatedTimestamp`.